osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -151,7 +151,7 @@ private void onSearchFinished(BeatmapListingFilterControl.SearchResult searchRes
                 }
 
                 // spawn new children with the contained so we only clear old content at the last moment.
-                var content = new FillFlowContainer<BeatmapCard>
+                var content = new ReverseChildIDFillFlowContainer<BeatmapCard>
```

Comment:
Inline comment appreciated mentioning while `ReverseChildID` is required.

Reply:
Should that comment be added to each of the places that now use reverse flow?

Reply:
Probably best.. else I can imagine it getting removed and regressing the display.